systemtest/src/main/java/io/strimzi/systemtest/resources/operator/SetupClusterOperator.java

```diff
@@ -381,13 +382,26 @@ public void updateSubscription(OlmConfiguration olmConfiguration) {
     private void createClusterOperatorNamespaceIfPossible() {
         if (this.isClusterOperatorNamespaceNotCreated()) {
             cluster.setNamespace(namespaceInstallTo);
-            cluster.createNamespaces(CollectorElement.createCollectorElement(testClassName, testMethodName), namespaceInstallTo, bindingsNamespaces);
-            StUtils.copyImagePullSecrets(namespaceInstallTo);
 
-            extensionContext.getStore(ExtensionContext.Namespace.GLOBAL).put(Constants.PREPARE_OPERATOR_ENV_KEY + namespaceInstallTo, true);
+            if (this.extensionContext != null) {
+                ResourceManager.STORED_RESOURCES.computeIfAbsent(this.extensionContext.getDisplayName(), k -> new Stack<>());
+                ResourceManager.STORED_RESOURCES.get(this.extensionContext.getDisplayName()).push(
+                    new ResourceItem<>(this::deleteClusterOperatorNamespace));
+
+                cluster.createNamespaces(CollectorElement.createCollectorElement(testClassName, testMethodName), namespaceInstallTo, bindingsNamespaces);
+                StUtils.copyImagePullSecrets(namespaceInstallTo);
+
+                this.extensionContext.getStore(ExtensionContext.Namespace.GLOBAL).put(Constants.PREPARE_OPERATOR_ENV_KEY + namespaceInstallTo, true);
+            }
         }
     }
 
+    private void deleteClusterOperatorNamespace() {
+        LOGGER.info("Deleting of Namespace " + this.namespaceInstallTo);
```

Comment:
Use the same pattern as we use for other logs

Reply:
Like this?
```java
LOGGER.info("Deleting Namespace {}", this.namespaceInstallTo);
```